prime-router/metadata/HL7/catchall/hl7/segments/PRT/Observation-Location.yml

```diff
@@ -0,0 +1,60 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+# TODO need message level conditions to be able to implement FHIR > HL7 mapping
+
+# This segment does not appear in NIST or RADx MARS
+
+resourceType: Observation
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
```

Comment:
Status for this and PRT/PractitionerRole should probably be done the same as PRT/Device